components/dashboard/conf/Caddyfile

```diff
@@ -0,0 +1,45 @@
+{
+	auto_https off
+	admin off
+}
+
+(compression) {
+	encode zstd gzip
+}
+
+(caching) {
+	header Cache-Control "public, max-age=604800, must-revalidate"
+}
+
+(default) {
+	header -Server
+}
+
+(discard_log) {
+	log {
+		output discard
+	}
+}
+
+:80 {
+	import caching
+	import compression
+	import default
+	import discard_log
+
+	redir /.well-known/security.txt https://www.gitpod.io/.well-known/security.txt permanent
+	redir /environment-variables /settings/ permanent
+	redir /login-success / permanent
+
+	root * /www
+	file_server
+	try_files {path} /index.html
```

Comment:
is that an drop-in replacement for `try_files $uri $uri/ /index.html;`?

Reply:
Yes

Reply:
don't think so. 

we used to have `/foo` been tried as `/foo/` as well. 

would that be valid syntax? `try_files {path} {path}/ /index.html`

Reply:
Yes. https://caddyserver.com/docs/caddyfile/directives/try_files